client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -613,15 +599,17 @@ const SubmissionEditStepForm = (props) => {
           }}
         />
         {attempting && (
-          <>
+          <div className="flex flex-nowrap">
             {renderResetButton()}
             {renderSubmitButton()}
             {renderContinueButton()}
-          </>
+            <Box sx={{ flex: '1', width: '100%' }} />
+            {question.type === questionTypes.Programming &&
```

Comment:
Leave a TODO: here to update the logic for `renderGetLiveFeedbackButton` pending #7418